### DISCUSSION TITLE
OpenTelemetry java agent different export time interval for different metrics

### DISCUSSION THREAD

**Discussion by kristinaskopinceva** [open-telemetry/community#1856] [Community Discussions]
hello everyone! 
is it possible set different export time interval for different metrics? In documentation contains about Environment variable - OTEL_METRIC_EXPORT_INTERVAL, but this interval is one on project. 
Use java agent.

**Comment by trask** [comment]
hi @kristinaskopinceva! I'd recommend asking in https://github.com/open-telemetry/opentelemetry-java-instrumentation
